_notebooks/2021-02-09-task-spooler.ipynb

```diff
@@ -0,0 +1,2746 @@
+{
```

Comment:
<p>can we make this more short ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LibreCV/blog/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p><code>#collapse-hide</code>  here too?</p><p> </p>

Reply:
In case you wanna check, it's pretty simple to run locally

https://github.com/LibreCV/blog/blob/master/_fastpages_docs/DEVELOPMENT.md#basic-usage-viewing-your-blog

Reply:
<p>can drop this output completely, it does not tell much and rather how the loss in chart</p>

Reply:
I just tested in my local machine and `#collapse-hide` doesn't seem to work at all.

Reply:
@edgarriba `#colapse-output` works nicely on my local build. But following @Borda's comment, I will just `%%capture` all the outputs.

Reply:
@edgarriba btw `#collapse-hide` seems to hide only codes.

Reply:
great